x/ibc/ante/ante_test.go

```diff
@@ -169,20 +174,19 @@ func (suite *HandlerTestSuite) TestHandleMsgPacketOrdered() {
 	suite.Error(err, "%+v", err) // invalid proof
 
 	suite.updateClient()
-	cctx, _ = suite.ctx.CacheContext()
```

Comment:
Note for @fedekunze - this test was wrong, since the initial recv sequence is 1 when the client is created.

Reply:
@mossid lol